apps/web/src/lang/fr.json

```diff
@@ -5382,6 +5386,10 @@
     "defaultMessage": "Le droit de priorité est un statut offert par la Commission de la fonction publique du Canada. Pour en apprendre davantage, <priorityEntitlementLink>consultez le site Web Information sur les droits de priorité</priorityEntitlementLink>.",
     "description": "Sentence describing what priority entitlement is"
   },
+  "Wpj2OY": {
+    "defaultMessage": "\"Je suis membre d’une Première Nation\"",
```

Comment:
@esizer Shouldn't this use _guillemets_ instead of quotes?

`"defaultMessage": "« Je suis membre d’une Première Nation »",`

Reply:
Oh probably, that is just how I received it from translation services 🤷‍♀️